rfcs/067-prismic-api-ids.md

```diff
@@ -0,0 +1,37 @@
+# RFC 067: Prismic API ID casing
+
+## TL;DR
+- Use kebab-case for Custom type API IDs (plural/singular for reusable/single respectively)
+- Use camelCase for all (Custom type and Slice) field API IDs
+- Use snake_case for Slice API IDs
+
+## Background
+We’d like to make sure our Prismic API IDs are written with consistent casing and that we’re following defaults from Prismic so that we’re going with the grain as far as possible and not having to manually override auto-generated SliceMachine files.
+
+## Custom type API IDs
+These should be kebab-cased and plural when the type is ‘reusable’ as opposed to ‘single’ (e.g. the reusable ‘Exhibition highlight tour’ Custom type has the API ID `exhibition-highlight-tours`, whereas the single ‘Global alert’ Custom type has the API ID `global-alert`).
+
+SliceMachine makes these snake_case by default but allows this to be overridden easily in the GUI - we've done this fairly consistently although a couple of reusable content types have been given singular API IDs – `card` and `collection-venue`. I don't think there's any mileage in updating everything to snake_case since *none* of them are in that format currently. I'm not sure if we should try to update to `cards` and `collection-venues` or just live with those as they are?
+
+## Field API IDs
+We have a mixture of snake_case and camelCase field API IDs. SliceMachine defaults to snake_case (based on the label), although this is readily overridable in the GUI at the point of adding the field.
```

Comment:
Think the acronym would ideally be spelled out fully on first mention (think onboarding etc)